.config/nextest.toml

```diff
@@ -1,2 +1,5 @@
 [profile.default]
 slow-timeout = { period = "60s", terminate-after = 3, grace-period = "30s" }
+
+[profile.windows]
+retries = { backoff = "exponential", count = 3, delay = "10s", jitter = true }
```

Comment:
Does the default profile also activate on windows profile? `slow-timeout` is necessary to avoid a test case hanging forever.

Reply:
Good catch! I think the default profile can also benifit with the "retries" as well. Merge the two together to one default, WDYT?

Reply:
👌